frontend/src/components/Order/Order.js

```diff
@@ -1,88 +1,78 @@
-import React, { useEffect, useState } from "react";
+import React, { useState } from "react";
 import { Card, Accordion, Button } from "react-bootstrap";
-import ReviewModal from "../Dashboard/reviewModal";
+import ReviewModal from "./ReviewModal";

```

Comment:
Maybe leave ReviewModal in Dashboard folder since it is a component on the dashboard.

Reply:
In this version, Modal is still part of the Order, will move it once it is part of Dashboard.